index.html

```diff
@@ -7,7 +7,7 @@
   <title>Welcome to Potionfy<title>
 </head>
 <body>
-
+	<h1>Welcome to Potionfy. Enjor your stay. </h1>
```

Comment:
Could you remove the 'Enjoy your stay.' string and make the 'Welcome to Potionfy' string red?